app/models/ability.rb

```diff
@@ -29,6 +29,6 @@ def default_abilities(user)
 
     can :read, Community
     can :read, Index, hidden: false
-    can :hide, Index if user.index_admin?
+    can :hide, Index, published?: true if user.index_admin?
```

Comment:
I think we have a problem then, unless I'm missing something in the cancancan docs or I'm just not seeing how to write the hash for !nil.

What I think we want (this might be where I'm getting hung up and not seeing the solution) is `can :hide, Index, published: !nil if user.index_admin?`

!nil is not valid in that context though, which is why I added the method.  Looking at it more though, it seems to expect right side of the key-value to be a scope or model attribute which is why accesible_by fails.  

Next thing I tried was using the block syntax, but accessible_by cannot function on abilities that use the block syntax. 

So:
```
Index.accessible_by needs to act on something on the Model
can :hide, @index, needs to act on something on an instance of the Model. 
```
Is that correct?  If it is, is the solution to add a published boolean?